server/tests/io/feed_parsers/belga_newsml_1_2_test.py

```diff
@@ -19,15 +19,21 @@
 
 class BelgaNewsMLOneTestCase(TestCase):
     filename = 'belga_newsml_1_2.xml'
+    filename_belga_remote = 'belga_remote_newsml_1_2.xml'
```

Comment:
Just a small note: I think it's better to create a new 
```
BelgaRemoteNewsMLOneTestCase(BelgaNewsMLOneTestCase)
``` 
class and inherit it from `BelgaNewsMLOneTestCase`, so you can simply overwrite `filename` with the file you want to use. For now, it's fine, but if we need to add more tests in the future, the current approach will slow down things. For example, redundant file reading, parsing and etc is happening when `test_can_parse` and `test_content` run. We don't really need to read and parse `belga_remote_newsml_1_2.xml` if we test only parsed data from `belga_newsml_1_2.xml`.
It's 5 mins fix, but IMO is better to make it.